docs/source/experiment_tracking.md

```diff
@@ -35,11 +35,11 @@ The choice of experiment tracking tool depends on your use case and choice of co
 - **MLflow** - You can combine MLflow with Kedro by using [`kedro-mlflow`](https://kedro-mlflow.readthedocs.io/en/stable/) if you require experiment tracking, model registry and/or model serving capabilities or have access to Managed MLflow within the Databricks ecosystem.
 - **Neptune** - If you require experiment tracking and model registry functionality, improved visualisation of metrics and support for collaborative data science, you may consider [`kedro-neptune`](https://docs.neptune.ai/integrations/kedro/) for your workflow.
 
-[We support a growing list of integrations](https://docs.kedro.org/en/stable/extend_kedro/plugins.html).
+{doc}`We support a growing list of integrations<kedro:extend_kedro/plugins>`.
 
 ## Set up a project
 
-This section describes the steps necessary to set up experiment tracking and access logged metrics, using the [spaceflights tutorial](https://docs.kedro.org/en/stable/tutorial/spaceflights_tutorial.html) with a version of Kedro equal to or higher than 0.18.4, and a version of Kedro-Viz equal to or higher than 5.2.
+This section describes the steps necessary to set up experiment tracking and access logged metrics, using the {doc}`spaceflights tutorial<kedro:tutorial/spaceflights_tutorial>` with a version of Kedro equal to or higher than 0.18.4, and a version of Kedro-Viz equal to or higher than 5.2.
```

Comment:
I noticed that all these `{doc}` references are opened in a new window, is that on purpose?

Reply:
It wasn't 😆 although I'm asked for it often enough and I think it's quite a reasonable thing to do. How to control this?

Reply:
It's a good question 😅 I'll do some digging